gateway/cli/src/main.rs

```diff
@@ -316,6 +324,12 @@ async fn main() -> anyhow::Result<()> {
                     })
                     .await?;
             }
+            LightningCommands::CloseChannelsWithPeer { pubkey } => {
+                let num_channels_closed = client()
+                    .close_channels_with_peer(CloseChannelsWithPeerPayload { pubkey })
+                    .await?;
+                println!("Closed {num_channels_closed} channel(s)");
```

Comment:
Ick. Can we change this to return JSON with `num_channels_closed` as one of the fields? That way tests can parse it and we can use `print_response` here instead

Reply:
Totally missed that :grima